src/article/editor/DownloadSupplementaryFileTool.js

```diff
@@ -6,7 +6,6 @@ export default class DownloadSupplementaryFileTool extends Tool {
     let el = super.render($$)
     el.append(
       $$('a').ref('link')
-        .attr('target', '_blank')
```

Comment:
We need to do a platform check here, because the browser version does need this.

Reply:
I have changed and committed this.